encoding/toml.ts

```diff
@@ -528,11 +529,11 @@ class Dumper {
 export function stringify(srcObj: object): string {
   let out: string[] = [];
   out = new Dumper(srcObj).dump();
-  return out.join("\n");
+  return out.join(EOL.LF);
 }
 
 export function parse(tomlString: string): object {
   // File is potentially using EOL CRLF
-  tomlString = tomlString.replace(/\r\n/g, "\n").replace(/\\\n/g, "\n");
+  tomlString = format(tomlString, EOL.LF).replace(/\\\n/g, EOL.LF);
```

Comment:
I would just remove this line entirely. The parser should be able to handle either line ending.

Reply:
If it's not here the parser will do it in itself. Example in golang csv reader: https://github.com/golang/go/blob/master/src/encoding/csv/reader.go#L236